policies/master_payer.json

```diff
@@ -29,16 +29,28 @@
     },
     {
       "Action": [
-        "aws-portal:View*",
+        "account:GetAccountInformation",
+        "billing:Get*",
         "budgets:Describe*",
         "budgets:View*",
-        "ce:Get*",
         "ce:Describe*",
+        "ce:Get*",
         "ce:List*",
+        "consolidatedbilling:Get*",
+        "consolidatedbilling:List*",
         "cur:Describe*",
-        "pricing:*",
+        "cur:Get*",
+        "cur:Validate*",
+        "freetier:Get*",
+        "invoicing:Get*",
+        "invoicing:List*",
         "organizations:Describe*",
-        "organizations:List*"
+        "organizations:List*",
+        "payments:Get*",
+        "payments:List*",
+        "pricing:*",
+        "tax:Get*",
+        "tax:List*"
       ],
       "Resource": "*",
       "Effect": "Allow",
```

Comment:
Should the Policies "Sid" be updated to reflect the change from CZCostMonitoring20210423 to CZCostMonitoring20230118?

Reply:
yes